llama_index/callbacks/global_handlers.py

```diff
@@ -22,6 +22,23 @@ def create_global_handler(eval_mode: str, **eval_params: Any) -> BaseCallbackHan
         handler: BaseCallbackHandler = WandbCallbackHandler(**eval_params)
     elif eval_mode == "openinference":
         handler = OpenInferenceCallbackHandler(**eval_params)
+    elif eval_mode == "openinferencetracer":
```

Comment:
Should we name this `arize_phoenix` since it runs Phoenix in the background?

Reply:
agreed, it's really an `arize_phoenix` one-click

Reply:
changed